src/main/java/ua/ithillet/lesson6/WorkWithListCollection.java

```diff
@@ -0,0 +1,69 @@
+package ua.ithillet.lesson6;
+
+import java.util.*;
+
+public class WorkWithListCollection {
+
+    public static int countOccurrence(List<String> list, String searchItem) {
+        int count = 0;
+        for (String item : list) {
+            if (item.equals(searchItem)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List<Integer> toList(ArrayList<Integer> list) {
+        List<Integer> targetList = new ArrayList<>(list);
+        return targetList;
+
+    }
+
+    public static List<Integer> findUnique(List<Integer> list) {
+        List<Integer> uniqueList = new ArrayList<>();
+        try {
+            for (int i : list) {
```

Comment:
```suggestion
            for (Integer i : list) {
```
список дозволяє null, а якщо autoboxing (автоматична конвертація з Integer в int), то з null значенням буде NullPointerException